src/plugins/1/accessors/sdk/index.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: full.html.hbs
+title: sdk
+---
+
+# sdk
+
+{{{since "1.6.0"}}}
+
+Accessor to the embedded SDK. 
+
+The embedded SDK is a custom version of our [Javascript SDK]({{ site_base_path }}sdk-reference/js/6) that uses a custom protocol plugged directly into Kuzzle core.  
+
+Plugin developers can use every controllers except the `realtime` controller and also the [query]({{ site_base_path }}sdk-reference/js/6/kuzzle/query/) method of the SDK.
```

Comment:
Your sentence suggests that the `query` method cannot be used by plugin developers, but that's not the case.

```suggestion
All the documented controllers can be used, except the `realtime` one.  
Also, the low-level [query]({{ site_base_path }}sdk-reference/js/6/kuzzle/query/) method is available for use.
```